src/player/client/pages/Ted/index.jsx

```diff
@@ -107,44 +115,125 @@ export default function Ted(props) {
           <Select
             value={filters.topics}
             style={{ minWidth: 150 }}
-            onChange={(e) => setFilters((old) => ({ ...old, topics: e }))}
+            onChange={(e) => {
+              setFilters((old) => ({ ...old, topics: e }));
+              ga.event("choosing_topic", "click").send();
+            }}
             placeholder="Topics"
           >
-            <Option value="">All</Option>
+            <Option value="" onClick={ga.event("topic_is_All", "click").send()}>
```

Comment:
Nooooo !! why do you call a function when mounting a component?
```suggestion
            <Option value="" onClick={()=>ga.event("topic_is_All", "click").send()}>
```